sql/src/test/java/io/crate/analyze/CreateAlterPartitionedTableAnalyzerTest.java

```diff
@@ -311,10 +311,28 @@ public void testAlterPartitionedTableShards() {
     }
 
     @Test
-    public void testAlterTablePartitionWithNumberOfShardsIsInvalid() {
-        expectedException.expect(IllegalArgumentException.class);
-        expectedException.expectMessage("Invalid property \"number_of_shards\" passed to [ALTER | CREATE] TABLE statement");
-        e.analyze("alter table parted partition (date=1395874800000) set (number_of_shards=1)");
+    public void testAlterTablePartitionWithNumberOfShards() {
+        AlterTableAnalyzedStatement analysis = e.analyze(
+            "alter table parted partition (date=1395874800000) set (number_of_shards=1)");
+        assertThat(analysis.partitionName().isPresent(), is(true));
+        assertThat(analysis.table().isPartitioned(), is(true));
+        assertEquals("1", analysis.tableParameter().settings().get(TableParameterInfo.NUMBER_OF_SHARDS.getKey()));
+    }
+    public void testAlterTablePartitionSetShards() {
```

Comment:
isn't this a duplicate of `testAlterTablePartitionWithNumberOfShards`?

Reply:
Looks like it, I'll remove it unless @smakalias had some different intention for it?

Reply:
it's duplicate, can't really remember whether I wanted to add another test or if it's copy paste glitch. It's OK to remove - sorry